test/unit/sql/SqlService.js

```diff
@@ -173,44 +173,44 @@ describe('SqlServiceTest', function () {
         it('should construct a SqlResultImpl with default result type if it\'s not specified', async function () {
             const fake = sandbox.replace(SqlResultImpl, 'newResult', sandbox.fake(SqlResultImpl.newResult));
             await sqlService.execute('s', [], { cursorBufferSize: 1 });
-            fake.calledOnceWithExactly(
-                sqlService,
-                fakeSerializationService,
-                fakeConnection,
-                fakeQueryId,
-                1,
-                SqlServiceImpl.DEFAULT_FOR_RETURN_RAW_RESULT,
-                fakeClientUUID
+            fake.calledWith(
+                sandbox.match(sqlService),
+                sandbox.match.func,
+                sandbox.match(fakeConnection),
+                sandbox.match(fakeQueryId),
+                sandbox.match(1),
+                sandbox.match(SqlServiceImpl.DEFAULT_FOR_RETURN_RAW_RESULT),
+                sandbox.match(fakeClientUUID)
             ).should.be.true;
         });
 
         it('should construct a SqlResultImpl with default cursor buffer size if it\'s not specified', async function () {
             const fake = sandbox.replace(SqlResultImpl, 'newResult', sandbox.fake(SqlResultImpl.newResult));
 
             await sqlService.execute('s', [], { returnRawResult: true });
-            fake.calledOnceWithExactly(
-                sqlService,
-                fakeSerializationService,
-                fakeConnection,
-                fakeQueryId,
-                SqlServiceImpl.DEFAULT_CURSOR_BUFFER_SIZE,
-                true,
-                fakeClientUUID
+            fake.calledWith(
+                sandbox.match(sqlService),
+                sandbox.match.func,
+                sandbox.match(fakeConnection),
+                sandbox.match(fakeQueryId),
+                sandbox.match(SqlServiceImpl.DEFAULT_CURSOR_BUFFER_SIZE),
+                sandbox.match(true),
+                sandbox.match(fakeClientUUID)
             ).should.be.true;
         });
 
         it('should construct a SqlResultImpl with parameters passed', async function () {
             const fake = sandbox.replace(SqlResultImpl, 'newResult', sandbox.fake(SqlResultImpl.newResult));
 
             await sqlService.execute('s', [], { returnRawResult: true, cursorBufferSize: 1 });
-            fake.calledOnceWithExactly(
```

Comment:
This and similar 2 changes needed because it was failing when I wrote `sqlService.deserializeRowValue.bind(sqlService)` I think somehow bound methods differ from each other. Therefore I used calledWith with matchers. The purpose of this test is to assert default value is used, so there is no problem with the current approach.